environments/aat.yml

```diff
@@ -128,6 +128,7 @@ cname:
   - name: "immigration-appeal"
     ttl: 3600
     record: "hmcts-aat.azurefd.net"
+    shutter: true
```

Comment:
the shutter solution isn't deployed in aat, only in sandbox and prod, this will just set up a dns record that doesn't point to anything

Reply:
👍 Will create another one for Prod when ready to do it for real.

Reply:
@timja Sorry this is the real one for PROD, can you please review?